swagger1/src/main/java/com/cerner/beadledom/swagger/SwaggerUiResource.java

```diff
@@ -44,11 +45,12 @@
   @GET
   @Path("/ui")
   @Produces(MediaType.TEXT_HTML)
-  public StreamingOutput getSwaggerUi(@Context UriInfo uriInfo) {
+  public StreamingOutput getSwaggerUi(@Context UriInfo uriInfo, @Context SecurityContext securityContext) {
+    String httpSchema = securityContext.isSecure() ? "https" : "http";
```

Comment:
I think `Schema` should be `Scheme`.

Reply:
https://github.com/cerner/beadledom/pull/137/commits/6d4a428dd8851add16eafafe2d16fa87866c04bc